onboarding.md

```diff
@@ -0,0 +1,591 @@
+@Override
+public void initialize(ServiceExtensionContext context){
+        // ...
+        registerDataEntries(context);
+        // ...
+        }
+
+private void registerDataEntries(ServiceExtensionContext context){
+        var metadataStore=context.getService(MetadataStore.class);
+
+        GenericDataCatalogEntry file1=GenericDataCatalogEntry.Builder.newInstance()
+        .property("type","File")
+        .property("path","/path/to/assets/")
+        .property("filename","test-document.txt")
+        .build();
+
+        DataEntry entry1=DataEntry.Builder.newInstance().id("test-document").policyId(USE_EU_POLICY).catalogEntry(file1).build();
+        metadataStore.save(entry1);
+        }
+```
+
+This adds a `GenericDataCatalogEntry` to the `MetadataStore` (which is the catalog). Or, in other words, your provider
+now "hosts" one file named `test-documents.txt` located in the path `/path/to/assets/` on your development machine. It
+makes it available for transfer under its `id` `"test-document"`. While it makes sense to have some sort of similarity
+between file name and id, it is by no means mandatory.
+> **Please adjust this path as necessary to match your local environment!**
+
+Please also note that the registering of Policies is omitted from this document for clarity.
+
+The `FileTransferExtension` also registers a `FileTransferFlowController`, which is basically a glorified `cp` process
+with some additional checking Please note that the provider connector does *not* add the `api` module, so that means
+that no REST API is offered by the provider!
+In order to make use of the `FileTransferExtension`, we simply must add the correct dependency to the provider's build
+file:
+
+```kotlin
+implementation(project(":samples:04-file-transfer:transfer-file"))
+```
+
+### Create the "consumer" connector
+
+The consumer is the one "requesting" the data and providing a destination for it, i.e. a directory into which the
+provider can copy the requested file.
+
+While the consumer does not need the `transfer-file` module, it does indeed need the `api` module, which implements the
+REST API:
+
+```kotlin
+implementation(project(":samples:04-file-transfer:api"))
+```
+
+It is good practice to explicitly configure the consumer's API port in `consumer/config.properties` like we learned in
+previous chapters.
+
+### Add IDS modules
+
+Now we have a provider, that can perform a file transfer, and a consumer, that can accept commands via REST, but those
+two cannot yet "talk" to each other.
+
+The standard way of connector-to-connector communication is IDS. Thus, we must add the relevant IDS-module to both build
+files:
+
+```kotlin
+// in consumer/build.gradle.kts and provider/build.gradle.kts:
+implementation(project(":data-protocols:ids"))
+implementation(project(":data-protocols:ids:ids-policy-mock"))
+```
+
+This adds the entire IDS protocol suite to both connectors. Nothing more needs to be done here. We will see later how we
```

Comment:
the entire IDS protocl suite? already implemented!?

Reply:
does not say anywhere that everything is implemented already :) 
what it aims to show is add all the `:ids` modules can be added by just adding the "meta"-module (`ids/build.gradle.kts`). I adapted the text accordingly.